python/ray/autoscaler/v2/instance_manager/ray_installer.py

```diff
@@ -49,7 +51,13 @@ def install_ray(self, instance: Instance, head_node_ip: str) -> bool:
                 head_node_ip,
             ),
             setup_commands=with_head_node_ip(setup_commands, head_node_ip),
-            ray_start_commands=with_head_node_ip(ray_start_commands, head_node_ip),
+            ray_start_commands=with_envs(
+                ray_start_commands,
+                {
+                    "RAY_HEAD_IP": head_node_ip,
+                    "RAY_cloud_instance_id": instance.instance_id,
```

Comment:
@rickyyx Could you pls give a sample of ray start command with the new parameters, and show it in the description? Thanks.